src/main/java/core/basesyntax/figures/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.AreaCalculator;
+import core.basesyntax.DrawAble;
+
+public abstract class Figure implements AreaCalculator, DrawAble {
+    protected final String color;
```

Comment:
color should be Enum type variable, not String

Reply:
not resolved